requirements/common.txt

```diff
@@ -9,6 +9,9 @@ django-environ==0.8.1
 django-sites==0.11
 python-dotenv==0.20.0
 django-cors-headers==3.10.1
+django-uuid-upload-path==1.0.0
+django-uuid-upload-path==1.0.0
```

Comment:
@adarsh-pdgt I think we have defined `django-uuid-upload-path` package twice, let's fix this one.

Reply:
fixed